src/helpers/apify_helper.js

```diff
@@ -4,21 +4,27 @@ const { promisify } = require('util');
 const stream = require('stream');
 const { delayPromise } = require('apify-shared/utilities');
 const { ACT_JOB_TERMINAL_STATUSES } = require('apify-shared/consts');
+const path = require('path');
+const { saveJson } = require('./fs_helper');
+const {
+    STATE_OUT_FILE, DATA_DIR,
+    NAME_OF_KEBOOLA_INPUTS_STORE,
+} = require('../constants');
 
 const pipeline = promisify(stream.pipeline);
 
-const DEFAULT_POOLING_INTERVAL = 2000; // ms
+const DEFAULT_POOLING_INTERVAL = 5000; // ms
```

Comment:
This should be named `DEFAULT_POOLING_INTERVAL_MILLIS` instead of having unit in a comment